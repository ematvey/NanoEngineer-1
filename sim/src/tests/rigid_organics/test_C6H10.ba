LENGTH 0 1 1.52362626651
LENGTH 0 4 1.51238387984
LENGTH 0 6 1.0735669518
LENGTH 0 7 1.07358325248
LENGTH 1 2 1.53506840238
LENGTH 1 5 1.51717797242
LENGTH 1 8 1.07508418275
LENGTH 2 3 1.52613040072
LENGTH 2 4 1.5114658448
LENGTH 2 9 1.07382400793
LENGTH 3 5 1.51768178483
LENGTH 3 10 1.06897988756
LENGTH 3 11 1.07331915105
LENGTH 4 12 1.07490836819
LENGTH 4 13 1.07486231676
LENGTH 5 14 1.07190717882
LENGTH 5 15 1.07582758842
ANGLE 0 1 2 88.8417669897
ANGLE 0 1 5 115.31133268
ANGLE 0 1 8 116.329135913
ANGLE 0 4 2 90.143921574
ANGLE 0 4 12 112.182719923
ANGLE 0 4 13 116.965156484
ANGLE 1 0 4 90.3272701456
ANGLE 1 0 6 114.66058865
ANGLE 1 0 7 113.744576473
ANGLE 1 2 3 88.8357175653
ANGLE 1 2 4 89.9271765161
ANGLE 1 2 9 122.978644463
ANGLE 1 5 3 89.8148441482
ANGLE 1 5 14 112.645592676
ANGLE 1 5 15 116.140723833
ANGLE 2 1 5 90.1112718304
ANGLE 2 1 8 123.352869195
ANGLE 2 3 5 90.4313958352
ANGLE 2 3 10 113.853638985
ANGLE 2 3 11 114.70651811
ANGLE 2 4 12 112.118469886
ANGLE 2 4 13 116.315961489
ANGLE 3 2 4 115.099746204
ANGLE 3 2 9 116.050554342
ANGLE 3 5 14 112.419062638
ANGLE 3 5 15 117.241592183
ANGLE 4 0 6 114.495239609
ANGLE 4 0 7 115.611165213
ANGLE 4 2 9 118.177822551
ANGLE 5 1 8 117.386464291
ANGLE 5 3 10 114.947299102
ANGLE 5 3 11 114.344923244
ANGLE 6 0 7 107.534951052
ANGLE 10 3 11 107.990408083
ANGLE 12 4 13 108.316081314
ANGLE 14 5 15 107.881582998
