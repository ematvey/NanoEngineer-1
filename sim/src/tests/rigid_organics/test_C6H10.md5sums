stdout 967EC9C0F7895EA82287B95C9B8C2A74
stderr D41D8CD98F00B204E9800998ECF8427E
