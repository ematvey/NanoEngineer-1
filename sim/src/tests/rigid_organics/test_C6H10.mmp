mmpformat 050502 required; 050706 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (6.466723) (-0.000000, -0.000000, -0.000500) (1.000000)
egroup (View Data)
group (C4H10)
info opengroup open = True
mol (C4H10.pdb) def
atom 1 (6) (1127, -782, 34) def
atom 2 (6) (-420, 162, 1268) def
bond1 1
atom 3 (6) (-4, 1030, -433) def
bond1 2
atom 4 (6) (-1066, -122, -565) def
bond1 3
atom 5 (6) (1522, -686, -1201) def
bond1 1 3
atom 6 (6) (-1238, -685, 962) def
bond1 2 4
atom 7 (1) (1833, -910, 1342) def
bond1 1
atom 8 (1) (1732, -2282, 77) def
bond1 1
atom 9 (1) (282, -108, 1779) def
bond1 2
atom 10 (1) (-160, 1462, -375) def
bond1 3
atom 11 (1) (-1915, -517, -1614) def
bond1 4
atom 12 (1) (-2237, 683, -1231) def
bond1 4
atom 13 (1) (2234, 1201, -1069) def
bond1 5
atom 14 (1) (1291, -356, -1928) def
bond1 5
atom 15 (1) (-2386, -474, 1389) def
bond1 6
atom 16 (1) (-1217, -1716, 624) def
bond1 6
egroup (C4H10)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part C4H10

