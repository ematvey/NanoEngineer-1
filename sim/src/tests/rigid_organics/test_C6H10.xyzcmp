16
C6H10.mmp
C 1.229000 -0.793000 0.363000
C -0.006000 -0.037000 0.837000
C 0.015000 0.758000 -0.476000
C -1.226000 -0.009000 -0.924000
C 1.333000 0.115000 -0.842000
C -1.335000 -0.655000 0.445000
H 2.068000 -0.744000 1.031000
H 1.046000 -1.829000 0.149000
H 0.059000 0.422000 1.807000
H -0.054000 1.829000 -0.512000
H -1.047000 -0.688000 -1.730000
H -2.063000 0.608000 -1.190000
H 2.155000 0.803000 -0.762000
H 1.381000 -0.358000 -1.806000
H -2.155000 -0.267000 1.016000
H -1.387000 -1.729000 0.480000
