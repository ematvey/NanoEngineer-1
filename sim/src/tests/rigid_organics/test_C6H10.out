======= test_C6H10.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 10
== structure comparison ==
cutover to fine tolerance at 1
